app/config/index.js

```diff
@@ -35,4 +35,4 @@ if (error) {
 value.dbConfig = dbConfig
 value.messageQueueConfig = messageQueueConfig
 
-module.exports = value
+export default value
```

Comment:
aah, while you are here I'm not a fan of the way these config modules export the returned value from Joi's validation rather than the config itself.  As the value has a type of `any` but your editor should be able to infer the types from the literal config object.
<img width="362" alt="Screenshot 2024-11-13 at 1 58 46 pm" src="https://github.com/user-attachments/assets/38ebd440-53f3-4a72-9ee0-bf7e9b47f17a">
vs:
<img width="613" alt="Screenshot 2024-11-13 at 1 59 39 pm" src="https://github.com/user-attachments/assets/415bd64d-9085-42ca-a7c0-a5adedf42292">

Reply:
Yeah good catch - I'll change the export